subscription-engine/kafka-ws/app/Redis/RedisManager.js

```diff
@@ -16,12 +16,10 @@ class RedisManager {
   constructor() {
     logger.info('Creating the RedisManager singleton...');
 
+    this.config = ConfigManager.getConfig(KAFKA_WS_CONFIG_LABEL).redis;
+
     // TODO: Implement "retry_strategy"
-    const redisOptions = {
-      host: redisConfig.host,
-      port: redisConfig.port,
-      db: redisConfig.database,
-    };
+    const redisOptions = { ...this.config };
```

Comment:
This is not necessary. You already have the object in the `this.config` variable.